modules/console/pages/flow-console.adoc

```diff
@@ -0,0 +1,114 @@
+include::{mod-loc}attributes.adoc[]
+ifndef::mod-loc[]
+:mod-loc: ./partials/
+endif::mod-loc[]
+//Category: flow-console
+// Type: assembly
+[id="skupper-console"] 
+= Using the Skupper console
+
+include::{mod-loc}preface.adoc[]
+
+The Skupper console provides data and visualizations of the traffic flow between Skupper sites.
+
+// Type: procedure
+[id="enabling-console"] 
+== Enabling the Skupper console
+
+By default, when you create a {skupper-name} site, you do not the Skupper console.
+
+When enabled, the Skupper console URL is displayed whenever you check site status using `skupper status`.
+
+.Prerequisites
+
+* A Kubernetes namespace where you plan to create a site
+
+.Procedure
+
+. Determine which site in your {service-network} is best to enable the console.
++
+--
+Enabling the console also requires that you enable the flow-collector component, which requires resources to process traffic data from all sites.
+You might locate the console using the following criteria:
+
+* Does the {service-network} cross a firewall?
+For example, if you want the console to be available only inside the firewall, you need to locate the flow-collector and console on a site inside the firewall.
+
+* Is there a site that processes more traffic than other sites?
+For example, if you have a _frontend_ component that calls a set of services from other sites, it might make sense to locate the flow collector and console on that site to minimize data traffic.
+
+* Is there a site with more or cheaper resources that you want to use?
+For example, if you have two sites, A and B, and resources are more expensive on site A, you might want to locate the flow collector and console on site B.
+--
+
+. Create a site with the flow collector and console enabled:
++
+----
+$ skupper init --enable-console --enable-flow-collector
```

Comment:
Maybe this is not part of this CR.
I am thinking that  using 2 flag options means using them independently and that is not the case for the console. (example skupper init --enabled console it doesn't work). Probably it is more appropriate a new flag option called --enable-flow-collector-with-console
@ajssmith

Reply:
A good question for @ajssmith , but necessary in docs today